lib/ex_aws/timestream/write.ex

```diff
@@ -309,9 +309,13 @@ defmodule ExAws.Timestream.Write do
         |> camelize_keys(deep: false)
       end)
     end)
+    |> delete_empty_version()
```

Comment:
It looks like all of the parameters are optional, so would it make more sense to just do something like

`|> Enum.filter(fn {_, v} -> v == nil end)`

rather than treating `version` as a special case?

Reply:
Makes sense, updated PR to filter out all `Record` nil fields.
Thought about doing the same for dimensions, but they actually have two required fields.